helm/kyverno-policies-dx/templates/EnforceGSRegistries.yaml

```
@@ -0,0 +1,30 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-giantswarm-registries
+  annotations:
+    policies.kyverno.io/title: Enforce GiantSwarm registries
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Images from unknown, public registries can be of dubious quality and may not be
+      scanned and secured, representing a high degree of risk. Requiring use of known, approved
+      registries helps reduce threat exposure by ensuring image pulls only come from them. This
+      sample validates that container images only originate from the GiantSwarm registries.
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+  - name: validate-registries
+    match:
+      resources:
+        kinds:
+        - Pod
+    validate:
+      message: >-
+        Registry used is not a GiantSwarm registry
```

```suggestion
        Images must come from a trusted Giant Swarm registry.
```
Just stylistic. I like to word policy failures in a way that they indicate what the expected behavior should be.